src/main/webapp/css/main.css

```diff
@@ -161,6 +161,83 @@ body {
   margin-top: 10px;
 }
 
+.messageContainer{
+  width:100%;
```

Comment:
You only need the display flex here, and maybe the bottom margin, although that's barely visible because of the lateral distance.